java/spring/pom.xml

```diff
@@ -6,14 +6,14 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>[3.1,3.2)</version>
+		<version>3.1.5</version>
```

Comment:
why did you change this line ?

Reply:
Fixed versions offer stability and protect against unexpected changes, but now I see that in the whole project we used range. I will change it back to range

Reply:
Yes, I prefer using range to avoid too maintainance. The idea is to use the latest stable version of each framework

I still have issue for clojure (fixed version) r some jvm based framework that I need to exclude milestone (range in maven could takes milestones in addition of stable)